Example/Example/EditorDemoController.swift

```diff
@@ -849,9 +849,28 @@ extension EditorDemoController: TextViewAttachmentDelegate {
         }) 
         task.resume()
 
-        return Gridicon.iconOfType(.image)
+        return placeholderImage(forAttachment: attachment)
     }
-    
+
+    func textView(_ textView: TextView, placeholderForAttachment attachment: NSTextAttachment) -> UIImage {
+        return placeholderImage(forAttachment: attachment)
+    }
+
+    func placeholderImage(forAttachment attachment: NSTextAttachment) -> UIImage {
+        let placeholderImage: UIImage
+        switch attachment {
+        case _ as ImageAttachment:
+            placeholderImage = Gridicon.iconOfType(.image, withSize: CGSize(width:32, height:32))
```

Comment:
Since the 32x32 size seems to be standard, why not create a constant for the size.  Something like `defaultAttachmentSize` would do.